packages/nodes-base/nodes/Okta/UserDescription.ts

```diff
@@ -0,0 +1,524 @@
+import type { INodeProperties } from 'n8n-workflow';
+const BASE_API_URL = '/api/v1/users/';
+export const userOperations: INodeProperties[] = [
+	{
+		displayName: 'Operation',
+		name: 'operation',
+		type: 'options',
+		noDataExpression: true,
+		displayOptions: {
+			show: {
+				resource: ['user'],
+			},
+		},
+		options: [
+			// Create Operation
+			{
+				name: 'Create',
+				value: 'create',
+				description: 'Create a new user',
+				routing: {
+					request: {
+						method: 'POST',
+						url: BASE_API_URL,
+						qs: { activate: '={{$parameter["activate"]}}' },
+						returnFullResponse: true,
+					},
+				},
+				action: 'Create a new User',
+			},
+			// Delete Operation
+			{
+				name: 'Delete',
+				value: 'delete',
+				description: 'Delete an empty Bucket',
+				routing: {
+					request: {
+						method: 'DELETE',
+						url: '={{"/api/v1/users/" + $parameter["userId"]}}',
+						returnFullResponse: true,
+					},
+				},
+				action: 'Delete a user',
+			},
+			// Get Operation
+			{
+				name: 'Get',
+				value: 'get',
+				description: 'Get a user',
+				routing: {
+					request: {
+						method: 'GET',
+						url: '={{"/api/v1/users/" + $parameter["userId"]}}',
+						returnFullResponse: true,
+						qs: {},
+					},
+				},
+				action: 'Get a user',
+			},
+			// Get All Operation
+			{
+				name: 'Get Many',
+				value: 'getAll',
+				description: 'Get many users',
+				routing: {
+					request: {
+						method: 'GET',
+						url: BASE_API_URL,
+						qs: { q: '={{$parameter["searchQuery"]}}' },
+					},
+					send: {
+						paginate: true,
+					},
+				},
+				action: 'Get many users',
+			},
+			// Update Operation
+			{
+				name: 'Update',
+				value: 'update',
+				description: 'Update users',
+				routing: {
+					request: {
+						method: 'POST',
+						url: '={{"/api/v1/users/" + $parameter["userId"]}}',
+						qs: {
+							project: '={{$parameter["projectId"]}}',
+						},
+						returnFullResponse: true,
+					},
+				},
+				action: 'Update a user',
+			},
+		],
+		default: 'getAll',
+	},
+];
+
+export const userFields: INodeProperties[] = [
+	// Fields for 'create' and 'update' operations
+	{
+		displayName: 'First Name',
+		name: 'firstName',
+		type: 'string',
+		required: true,
```

Comment:
A lot of these fields are not `required` for the `update` operation

Reply:
fixed they are optional for update now